terraform/aws/modules/nginx-setup/README.md

```diff
@@ -0,0 +1,77 @@
+## Terraform & Shell Script for Nginx Setup with SSL
+
+## Overview
+This Terraform configuration script sets up a Nginx server with SSL certificates on an AWS EC2 instance.
+It fetches SSL certificates using Certbot and integrates with Kubernetes infrastructure from a specified GitHub repository
```

Comment:
also lets add the cron addition for renewal here.

Reply:
Have requested for few changes to @Mohanraj209, will take it in an another PR